tools/feature_matrix.py

```diff
@@ -82,6 +83,12 @@ class Feature(IntEnum):
     'safari': 140000,
     'node': 150000,
   },
+  Feature.MEMORY64: {
+    'chrome': 128,
```

Comment:
Is it premature to add this? we don't know which versions will ship this unflagged.

Reply:
Anyone targeting memory64 is already seeing an experimental warning, so that we are really targetting here is some arbitrary interim version of memory64.   Once we remove the experimental warning we will pin the final values here I imagine?